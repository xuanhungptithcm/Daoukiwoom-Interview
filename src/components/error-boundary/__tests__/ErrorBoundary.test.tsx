/**
 * @jest-environment jsdom
 */

import { shallow } from "enzyme";
import ErrorBoundary from "../ErrorBoundary";

describe("ErrorBoundary", () => {
  test("should properly render", () => {
    expect(shallow(<ErrorBoundary> abc</ErrorBoundary>)).toMatchSnapshot();
  });
});
