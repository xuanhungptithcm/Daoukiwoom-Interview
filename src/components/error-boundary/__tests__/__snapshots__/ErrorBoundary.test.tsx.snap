// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorBoundary should properly render 1`] = `" abc"`;
